lgc/include/lgc/state/ShaderStage.h

```diff
@@ -77,6 +77,11 @@ bool isShaderEntryPoint(const llvm::Function *func);
 // Gets name string of the abbreviation for the specified shader stage
 const char *getShaderStageAbbreviation(ShaderStage shaderStage);
 
+enum AddFunctionArgsFlag : unsigned {
```

Comment:
Maybe make this a class enum?

Reply:
That causes warnings/errors about the bit twiddling. If you have a not-too-boilerplatey solution for that, then sure :)